esmvalcore/experimental/recipe_info.py

```diff
@@ -150,12 +189,13 @@ def render(self, renderer: str = 'plaintext') -> str:
 class RecipeInfo():
```

Comment:
We will need to use `esmvalcore.Recipe` anyway to get the diagnostics / scripts, so I like the idea of inheriting from it or delegating to it. It would need a bit of refactoring to get this to work then.

 I think from a user point of view, it only makes sense to call it `Recipe` and call the `esmvalcore.Recipe` something like `RecipeDriver` or `RecipeEngine`.

Reply:
Let's make a new issue to refactor the esmvalcore.Recipe class, so that it will be easier to inherit from it in this API module.

Reply:
#934

Reply:
Didn't we agree to call this object a plain old 'Recipe' here?